lib/displayHelpers.ts

```diff
@@ -132,12 +133,16 @@ const checkAddress = (input: string, chainAddressPrefix: string) => {
     return error.toString();
   }
 
-  if (!prefix.startsWith(chainAddressPrefix)) {
-    return `Expected address prefix '${chainAddressPrefix}' but got '${prefix}'`;
+  if (chainAddressPrefix) {
+    if (!prefix.startsWith(chainAddressPrefix)) {
+      return `Expected address prefix '${chainAddressPrefix}' but got '${prefix}'`;
+    }
+  } else {
+    // any prefix is allowed
   }
```

Comment:
Why not `if(chainAddressPrefix && !prefix.startsWith(chainAddressPrefix))`?

Reply:
Yeah, this works too, sure. My idea was to have enabling of the check and the check itself separate. But no strong opinion here.